include/swift/Frontend/DiagnosticHelper.h

```diff
@@ -34,15 +34,13 @@ class DiagnosticHelper {
   /// OS is the stream to print diagnostics. useQuasiPID determines if using
   /// real PID when priting parseable output.
   static DiagnosticHelper create(CompilerInstance &instance,
+                                 const CompilerInvocation &invocation,
```

Comment:
What is the invocation here? I assume it's not the same as `instance.getInvocation()`?

Reply:
For the normal swift-frontend compilation, the Invocation is not moved into `instance` after the DiagnosticHelper is configured.